mmv1/products/netapp/Volume.yaml

```diff
@@ -521,6 +522,13 @@ properties:
         enum_values:
           - 'ENABLED'
           - 'PAUSED'
+      - name: 'hotTierBypassModeEnabled'
+        type: Boolean
+        description: |
+          Optional. Flag indicating that the hot tier bypass mode is enabled. Default is false.
+          Only applicable to Flex service level.
+        default_value: false
```

Comment:
Is the `default_value: false` required here? Does the API return `false` for this field if the default value is not provided in the provider and this field is not sent in the API request?